packages/wrapped-keys/src/lib/litActions/solana/src/signTransactionWithSolanaEncryptedKey.js

```diff
@@ -0,0 +1,60 @@
+const {
+  clusterApiUrl,
+  Connection,
+  Keypair,
+  Transaction,
+} = require('@solana/web3.js');
+
+(async () => {
+  const LIT_PREFIX = 'lit_';
+
```

Comment:
Let's add some check for user params passes in the jsPrams like in signTransactionWithEthereumEncryptedKeyLitAction

Reply:
Similar question [here](https://github.com/LIT-Protocol/js-sdk/pull/504/files#r1640572166), but the required params are:

```
jsParams: {
      accessControlConditions: decryptionAccessControlConditions,
      ciphertext,
      dataToEncryptHash,
      serializedTransaction,
      broadcast: false,
      solanaNetwork: 'devnet',
    }
```

Probably makes sense to check `solanaNetwork` is one of the three possible options (`devnet`, `testnet`, `mainnet-beta`). Of course `serializedTransaction` is serialized, so there are no tx params to check for

Reply:
Yes let's add a check for `solanaNetwork`

Reply:
`mainnet-beta`  is the actual mainnet